src/components/avatar/index.tsx

```diff
@@ -93,6 +124,20 @@ export type AvatarProps = Pick<AccessibilityProps, 'accessibilityLabel'> & {
    * fails (equiv. to Image.onError()).
    */
   onImageLoadError?: ImagePropsBase['onError'];
+  /**
+   * The name of the avatar user.
+   * Text initials will be generated from the name.
+   * Has lower priority than label.
+   */
+  name?: string;
+  /**
+   * Override the logic for creating initials of the user name (default from AvatarHelper)
+   */
+  getInitials?: (name?: string, limit?: number) => string;
+  /**
+   * Send this to use the name to infer a backgroundColor
+   */
+  nameToColorProps?: NameToColorProps;
```

Comment:
Maybe a better name can be colorHashConfig? autoColorsConfig? dyanmicColorsConfog? (:

Reply:
`autoColorsConfig` with `AutoColorsProps` 10x :)